ext_emconf.php

```diff
@@ -18,11 +18,11 @@
   'modify_tables' => '',
   'clearCacheOnLoad' => 0,
   'lockType' => '',
-  'version' => '1.0.2',
+  'version' => '1.2.0',
```

Comment:
The changes here and in the TYPO3 version requirement should be excluded from this pull request. I set those leading up to a release (and yes, the version is a bit behind in this file compared to tags).